core/templates/dev/head/components/create_button/CreateActivityButtonDirective.js

```diff
@@ -92,6 +94,7 @@ oppia.directive('createActivityButton', [function() {
         // editor if the create modal does not need to be shown).
         if (urlService.getUrlParams().mode === 'create') {
           if (!GLOBALS.can_create_collections) {
+            $location.path('created=true').replace;
```

Comment:
It appends created=true to the url so that the creation loop is prevented when the user presses the back button